_rules/SC2-4-2-page-has-title.md

```diff
@@ -34,7 +34,7 @@ The page contains at least one `title` element.
 
 ### Expectation 2
```

Comment:
This rule is also being worked over in: https://github.com/auto-wcag/auto-wcag/pull/440